tests/system/test_streams.py

```diff
@@ -114,6 +114,12 @@ def test_stream_stream_passing_dict(echo):
 
     @pytest.mark.asyncio
     async def test_async_unary_stream_reader(async_echo):
+        # TODO(https://github.com/googleapis/gapic-generator-python/issues/2152): Add test for async rest server-streaming.
```

Comment:
This issue link and any follow up links for server streaming within this file will be updated to:
https://github.com/googleapis/gapic-generator-python/issues/2168